src/views/catalog/CustomizeInstanceType/components/CustomizeITVMFooter.tsx

```diff
@@ -75,6 +82,10 @@ const CustomizeITVMFooter: FC = () => {
                       data: vmSignal.value || vm,
                       model: VirtualMachineModel,
                     });
+                    eventMonitor(CUSTOMIZE_VM_SUCCEEDED, {
```

Comment:
i can see in the rest of the PR that u always send a vmName and most of the time a flow, why not creating 3 funcs,

eventMonitorWithName , eventMonitorITFlow , eventMonitorTemplateFlow , the 2 last will use eventMonitorWithName, and u simply sending the VM and the key, all the rest of logic of adding a flow and a name will be under those 3 funcs, u wont need to import getName on each file, and simpler to work with

Reply:
Done! There were some places where the VM wasn't accessible, so I had to work around those.